src/common/sorted-table/index.js

```diff
@@ -307,6 +333,12 @@ export default class SortedTable extends Component {
     })
   }, 500)
 
+  _executeGroupedAction = createSelector(
```

Comment:
it's an error, a selector memoize the result of a function and return the cashed result when the variables passed in its params haven't changed.